src/java/edu/slu/tpen/servlet/LineServlet.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package edu.slu.tpen.servlet;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import static java.lang.Integer.parseInt;
+import java.sql.SQLException;
+import java.util.Map;
+import java.util.logging.Level;
+import static java.util.logging.Level.SEVERE;
+import java.util.logging.Logger;
+import static java.util.logging.Logger.getLogger;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import static javax.servlet.http.HttpServletResponse.SC_NOT_FOUND;
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import net.sf.json.JSONObject;
+import textdisplay.Folio;
+import utils.*;
+
+/**
+ *
+ * @author DrewSadler01
+ */
+public class LineServlet extends HttpServlet{
+    
+        /**
+     * Handles the HTTP <code>GET</code> method, returning a JSON-LD
+     * serialisation of the requested T-PEN canvas.
+     * If <code>Accept</code> header in the request is specified
+     * and contains <code>iiif/v3</code>, returns Presentation v3.0 serialisation.
+     * Otherwise, returns Presentation v2.1 serialisation.
+     *
+     * @param req servlet request
+     * @param resp servlet response
+     * @throws ServletException if a servlet-specific error occurs
+     * @throws IOException if an I/O error occurs
+     */
+    
+    //A callable method that would have the framework to resolve URI and a header request to the page
+    @Override
+    protected void doGetUri(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
```

Comment:
I think you are looking for [HTTPServlet's doGet](https://docs.oracle.com/javaee/7/api/javax/servlet/http/HttpServlet.html) here,  which you intend to `@override`